README.md

```diff
@@ -17,22 +18,28 @@ export enum EventDirection {
 
 ```
 
-**queueUrl**(required) SQS queue url
+**queueLink**(required) Prefix to get the queue details from environment variable, for example
```

Comment:
If we make this mandatory, then we drop the url. I was thinking they would be alternatives. If dropping it, then we shouldn't need AWS_REGION either

Reply:
right, I will update the readme